lib/krb5/krbhst.c

```diff
@@ -718,6 +728,13 @@ plugin_get_hosts(krb5_context context,
 {
     struct plctx ctx = { type, kd, 0 };
 
+    /*
+     * XXX Need a way to pass this through -- unsure if any of this is
```

Comment:
The plugins get the `krb5_context`, so they get to check this parameter.  I say let's trust them.